backend/routes/postReaction.py

```
@@ -25,207 +25,180 @@ def get_db():
-                    "user": {
-                        "id": reply.user.id,
-                        "username": reply.user.username,
-                        "profile_picture": f"http://127.0.0.1:8000/uploads/profile_pictures/{reply.user.profile_picture}"
-                    },
-                    "total_likes": len(reply.likes),
-                    "user_liked": any(like.user_id == current_user.id for like in reply.likes),
-                }
-                for reply in replies
-            ]
-        }
+    parents = db.query(Comment).filter(Comment.post_id == post_id, Comment.parent_id == None).all()
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)